appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -1395,6 +1397,8 @@ public void run(Objectify datastore) throws ObjectifyException {
 
           Preconditions.checkState(fd != null);
 
+          LOG.log(Level.INFO, "UploadRawFile: filename = " + fileName + " Role = " + fd.role);
```

Comment:
Since Google charges .50/GiB for logs we may not want to log every file in production.

Reply:
We have logging disabled on ai2, except for severe level entries

Reply:
But I'll remove it.